src/xrUICore/XML/UIXmlInitBase.cpp

```diff
@@ -742,7 +742,7 @@ bool CUIXmlInitBase::InitTabControl(CUIXml& xml_doc, pcstr path,
         newButton->m_btn_id = xml_doc.ReadAttrib("button", i, "id");
         if (!newButton->m_btn_id.size())
         {
-            R_ASSERT4(defaultIdsAllowed, "Tab control tab doesn't have 'id' assigned.", xml_doc.m_xml_file_name, path);
+            R_ASSERT4(ShadowOfChernobylMode || defaultIdsAllowed, "Tab control tab doesn't have 'id' assigned.", xml_doc.m_xml_file_name, path);
```

Comment:
This change effectively disables the crash, but doesn't fix it.
Tabs PDA still don't have an ID and don't work.